.gitlab/includes/clang11_pipeline.yml

```diff
@@ -19,77 +19,28 @@ include:
     CMAKE_FLAGS: "-DCMAKE_BUILD_TYPE=$BUILD_TYPE -DPIKA_WITH_CXX_STANDARD=$CXXSTD \
                   -DPIKA_WITH_MAX_CPU_COUNT=256 -DPIKA_WITH_MALLOC=system \
                   -DPIKA_WITH_SPINLOCK_DEADLOCK_DETECTION=ON"
+    CSCS_REBUILD_POLICY: always
```

Comment:
We don't want this everywhere do we? The pika build stage would be ok to always rebuild, but the compiler and dependencies not? Or does it fetch an already built image to avoid actually rebuilding, even if this says "always"?